MkChrootTree.sh

```diff
@@ -177,6 +177,8 @@ BINDSOURCES=( $(grep -v "${ALTROOT}" /proc/mounts | sed '{
                  /^none/d
                  /\/tmp/d
                  /rootfs/d
+		 /\/ /d
```

Comment:
Looks like this should be spaces instead of tabs, for consistency...

Reply:
Freaking autoindent converting blocks of spaces to tabs...